.github/workflows/project-codeflare-release.yml

```diff
@@ -170,3 +170,13 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.CODEFLARE_MACHINE_ACCOUNT_TOKEN }}
       shell: bash
+
+  trigger-odh-cfo-build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Repository Dispatch
+      uses: peter-evans/repository-dispatch@v2
```

Comment:
I feel like we should run workflows using a similar pattern to other workflow runners (i.e using the gh cli). Just to keep consistency in our workflows. wdyt @sutaakar